aim/aim_store.py

```diff
@@ -312,13 +320,25 @@ def _before_session_commit(session, flush_context, instances):
 
 class K8sStore(AimStore):
 
-    def __init__(self, namespace=None, config_file=None):
+    db_model_map = {api_res.VmmInjectedNamespace: api_v1.Namespace,
+                    api_res.VmmInjectedDeployment: api_v1.Deployment,
+                    api_res.VmmInjectedReplicaSet: api_v1.ReplicaSet,
+                    api_res.VmmInjectedService: api_v1.Service,
+                    api_res.VmmInjectedHost: api_v1.Node,
+                    api_res.VmmInjectedGroup: api_v1.Pod}
+
+    def __init__(self, namespace=None, config_file=None,
+                 vmm_domain=None, vmm_controller=None):
         super(K8sStore, self).__init__()
         self.klient = api_v1.AciContainersV1(config_file=config_file)
         self.namespace = namespace or api_v1.K8S_DEFAULT_NAMESPACE
+        self.attribute_defaults = {'domain_type': 'Kubernetes',
+                                   'domain_name': vmm_domain or 'kubernetes',
+                                   'controller_name':
+                                   vmm_controller or 'kube-cluster'}
```

Comment:
this store doesn't receive the config object, so it cannot rely on get the defaults from the config file. The defaults are provided for the sake of completeness of this class.